forge-core/src/main/java/forge/MulliganDefs.java

```diff
@@ -9,10 +9,40 @@
  */
 public class MulliganDefs {
     public enum MulliganRule {
-        Original,
-        Paris,
-        Vancouver,
-        London
+        Original {
+            @Override
+            public int getModifiedHandSize(int startingHandSize) {
+                return startingHandSize;
```

Comment:
SHouldn't this function just be in AbstractMulligan once and overridden once in your new Mulligan class?